### PR TITLE
Cleaning/category

### PR DESCRIPTION
**Pull request by andreaseliasson**

### CODE REVIEW

File: src/cleaning_functions.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import pandas as pd
+import re
```
Comment (arahayrabedian) [inline src/cleaning_functions.py]: minor PEP8 reordering recommended.

**Review by arahayrabedian — COMMENTED** [review]
import reordering if you want to be pep8 strict, otherwise LGTM

**Comment by andreaseliasson** [comment]
@arahayrabedian: fixed pep8 import issue.

**Review by arahayrabedian — APPROVED** [review]

**Comment by andreaseliasson** [comment]
@arahayrabedian: branch has been rebased on top of master and latest commit reflects the Oldmainmethod.py setup.

**Comment by arahayrabedian** [comment]
still looks good, haven't run it myself but eyeballing it looks nice.

**Comment by utkuozbulak** [comment]
Umm, can you help me understand something: 
Note from the first commit:
> For now this function is used to clean the category strings by
removing the 'Jobs' substring. The 'Jobs' is superfluous data which
appears in each category so we can remove `

If, before this conversion there are N unique categories, after this conversion there are same N different categories (removing 'jobs' from each one). Whats the point ? When we encode it having 'job' in the string doesn't matter since everything will be encoded.

Or, were there some categories such as 'Engineering' and 'Engineering Jobs' and with this cleaning we reduced the amount of unique categories ? Is this the case ?

**Review by utkuozbulak — CHANGES_REQUESTED** [review]
Questions questions

**Comment by andreaseliasson** [comment]
@utkuozbulak: I see your point. Removing 'Jobs' will not reduce the number of unique categories.

My reasoning for doing this was to reduce the amount of encoding that had to be done. Consequently reducing the amount of bits that had to be used and thus reducing space and possibly run-time.

But if the encoding process doesn't take into account the length of the categories, then we can discard the changes. If so, I will update the branch to only include the minor fixes (we have duplicate 'ContractTime' features).

**Comment by utkuozbulak** [comment]
Encoding(the way we do it now) doesn't take the amount of bits into account so the from that perspective it won't change but there is no need to discard the change. Let it stay like this, maybe we will end up changing the way we encode and then it might help. Who knows ?

**Review by utkuozbulak — APPROVED** [review]
Okidok

**Pull request merged**